tests/baselines/reference/user/webpack.log

```diff
@@ -4,27 +4,11 @@ lib/ContextModule.js(436,34): error TS2341: Property 'moduleGraph' is private an
 lib/ContextModule.js(465,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/ContextModule.js(767,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/Dependency.js(139,29): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
-lib/ExternalModule.js(284,54): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
+lib/ExternalModule.js(285,54): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
```

Comment:
Yayyyy, mostly fixed webpack. @ahejlsberg says this change is intentional from his contextual type change and it because there's an inaccurate annotation (binding pattern?) somewhere that's now applying.